styles/components/text.css

```diff
@@ -0,0 +1,9 @@
+.small-text{
+    font-size: 12px;
```

Comment:
Try to use consistent units for _font-size_ like _rem_ instead of mixing _px_ and _rem_